.github/workflows/main.yml

```
@@ -146,3 +146,28 @@ jobs:
             repo: 'rsconnect-jupyter',
             event_type: 'rsconnect_python_latest'
           })
+  test-rsconnect:
+    name: "Test vetiver"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
```

```suggestion
          python -m pip install --upgrade pip
          python -m pip install -r requirements.txt
```
I think we'll need to install all the requirements? I originally thought this wouldn't be necessary due to `actions/checkout@v2`, but it looks like the tests might need everything installed anyway. Hope this helps!